src/index.html

```diff
@@ -22,6 +22,402 @@
+              href="https://en.wikipedia.org/wiki/Laptop"
+              data-qa="nav-hover"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Gadget"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Tablet_computer"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Photograph"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__img">
+          <img
+            class="card__item"
+            src="./images/imac.jpeg"
+            alt="imac"
+          />
+        </div>
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <h3 class="card__code">Product code: 195434</h3>
+
+        <div>
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+        </div>
+
+        <div class="card__price">Price:</div>
+
+        <div class="card__buy">
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card__img">
+          <img
+            class="card__item"
+            src="./images/imac.jpeg"
+            alt="imac"
+          />
+        </div>
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <h3 class="card__code">Product code: 195434</h3>
+
+        <div>
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+        </div>
+
+        <div class="card__price">Price:</div>
+
+        <div class="card__buy">
+          <a
+            href="#"
+            class="card__button"
+            data-qa="hover"
```

Comment:
The value of the 'data-qa' attribute should be 'card-hover' to maintain consistency across the project, as seen in line 149.